Livecheckables/imageoptim-cli.rb

```diff
@@ -1,4 +1,3 @@
 class ImageoptimCli
-  livecheck :url   => "https://www.npmjs.com/package/imageoptim-cli",
-            :regex => %r{package__sidebarText.*?>([0-9\.]+)</p>}
+  livecheck :regex => /^(\d+(?:\.\d+)+)$/
```

Comment:
```suggestion
  livecheck :regex =>/^v?(\d+(?:\.\d+)+)$/
```

The generic regex I use for Git tags that have a format like `v1.2.3` or `1.2.3` (where that's the only text) is: `/^v?(\d+(?:\.\d+)+)$/`

The only difference is the optional `v?` on the front. Even if the tags don't ever have a `v` on the front, I still add the optional `v?` because sometimes people make a mistake and add a tag that doesn't follow suit.

If someone added a tag for `v1.2.3`, your regex wouldn't match it. I've seen this happen in upstream repos while working on livecheckables, so it's best to account for it even if it hasn't happened yet.